### ISSUE TITLE
mapping_fxn has no access to HOOMD box

### ISSUE THREAD

**Issue by RainierBarrett** [ur-whitelab/hoomd-tf#336]
In [tensorflowcompute.py](https://github.com/ur-whitelab/hoomd-tf/blob/3f2a11c1db60274dbc49afd68c49f67d0843b3be/htf/tensorflowcompute.py#L198), the `mapping_fxn` argument is presently unable to access information about the simulation box, only particle positions. In order to properly treat periodic boundary conditions, the box dimensions are needed.

**Comment by RainierBarrett** [comment]
This could be addressed in a couple of ways. Easiest is probably to change the documentation and call to `mapping_fxn` to expect it to take in the simulation box as well as positions.

**Comment by whitead** [comment]
Is the box not present in a snapshot?

**Comment by RainierBarrett** [comment]
Sure it is, but I'm talking about the `mapping_fxn` call expected signature, not the signature of `enable_mapped_nlist`. Inside of `enable_mapped_nlist` we say the expected signature of the mapping function is only 
```
:param mapping_fxn: a function whose signature is ``f(positions)`` where positions is an
                            ``Nx4`` array of fine-grained positions and
                            whose return value is an ``Mx4`` array
                            of coarse-grained positions.
:type mapping_fxn: python callable
```
so when we call it within `enable_mapped_nlist` we don't presently feed in box information:

```python
g_pos = mapping_fxn(
            snap.particles.position.astype(self.model.dtype))
```

I should be able to open a PR with a small change either today or tomorrow on this.

**Comment by whitead** [comment]
Got it, thanks

**Comment by whitead** [comment]
Closed by #337
